manager.go

```diff
@@ -692,36 +692,40 @@ func (m *jobManager) resolveImageOrVersion(imageOrVersion, defaultImageOrVersion
 		return unresolved, "", "", nil
 	}
 
-	imagestreams := make(map[string]string)
-	archSuffix := ""
+	type namespaceAndStream struct {
+		Namespace   string
+		Imagestream string
+		ArchSuffix  string
+	}
+
+	imagestreams := []namespaceAndStream{}
 	switch architecture {
 	case "amd64":
-		imagestreams["ocp"] = "release"
-		imagestreams["origin"] = "release"
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp", Imagestream: "release"})
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "origin", Imagestream: "release"})
 	case "arm64":
-		imagestreams["ocp-arm64"] = "release-arm64"
-		archSuffix = "-arm64"
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-arm64", Imagestream: "release-arm64", ArchSuffix: "-arm64"})
+	case "multi":
+		// the release-controller cannot assemble multi-arch release, so we must use the `art-latest` streams instead of `release-multi`
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.12-art-latest-multi", ArchSuffix: "-multi"})
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.11-art-latest-multi", ArchSuffix: "-multi"})
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.10-art-latest-multi", ArchSuffix: "-multi"})
+		imagestreams = append(imagestreams, namespaceAndStream{Namespace: "ocp-multi", Imagestream: "4.9-art-latest-multi", ArchSuffix: "-multi"})
```

Comment:
Is this something we're going to have to update every release?

Reply:
Yes. We will need to add the new nightly imagestreams when the become available since there isn't a single imagestream that tags them all, unlike the amd64 and arm64 release streams.

Reply:
Ack!  We might need to think about only adding the new entry when we bump the default version up (i.e. once we GA the current release) or we'll potentially be using an N and an N+1 version to launch.

Reply:
We might want to update the `lookup` command's output to handle the scenarios where more than 1 images is necessary to launch a cluster.

Reply:
We do not need 4.9/4.10 as it is not officially supported

Reply:
Removed the 4.9 and 4.10 imagestreams from multiarch